app/src/main/java/com/alphawallet/app/service/OkLinkService.java

```diff
@@ -123,6 +129,21 @@ public EtherscanEvent[] getEtherscanEvents(String address, TransferFetchType tfT
         return etherscanEvents.toArray(new EtherscanEvent[0]);
     }
 
+    //@SuppressWarnings("ConstantConditions")
+    private boolean compareEventsWithLastRead(List<OkxEvent> events, long lastBlockRead)
+    {
+        for (OkxEvent ev : events)
+        {
+            long height = Long.parseLong(ev.height);
```

Comment:
Might be a good idea to check if `ev.height` is null or empty (via `TextUtils.isEmpty()`) to avoid a potential crash